core/src/components/content/content.tsx

```diff
@@ -172,11 +172,13 @@ export class Content implements ComponentInterface {
   }
 
   private resize() {
-    if (this.fullscreen) {
-      readTask(() => this.readDimensions());
-    } else if (this.cTop !== 0 || this.cBottom !== 0) {
-      this.cTop = this.cBottom = 0;
-      forceUpdate(this);
+    if (Build.isBrowser) {
```

Comment:
And so I understand, this is to work around a stencil bug correct?

Reply:
This isn't a workaround per-say. The reported issues attached are resolved by this change.  

I had some discussions with Tanner around if `forceUpdate` should internally account for this. It checks for build conditions to see if it updatable: https://github.com/ionic-team/stencil/blob/main/src/runtime/update-component.ts#L328-L342, but should likely also check if it is in a browser context. At this time, we don't have enough historic data to confidently determine if `forceUpdate` should only execute in the browser or not. He is leading that conversation with the team and we will re-sync on the outcomes (if we can remove this in a future update of Stencil).

The problem that was occurring is that the pre-rendering logic would attempt to mock `ion-content` DOM tree, but would call `forceUpdate` as part of the first render, resulting in an infinite loop.

Reply:
Thanks! Can we include this context in the comment too?

Reply:
Also if there's an open stencil issue it would be good to link to that in the comment

Reply:
Added some extra context + created a spike in JIRA for Stencil to track (and referenced it in the code comments).